gen_vehicle_model.py

```diff
@@ -88,7 +88,7 @@
             args.vspec_file,
             include_dirs,
             merge_private=False,
-            break_on_noncore_attribute=strict,
+            break_on_unknown_attribute=strict,
```

Comment:
That property is a boolean value
```suggestion
            break_on_unknown_attribute=True,
```